### PR TITLE
#777

### PR DESCRIPTION
**Pull request by alisonailea**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [x] Edge (Win 10)
- ~~[ ] Unit tests written (`common` at minimum)~~
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- ~~[ ] One core team UX approval~~

### CODE REVIEW

File: src/components/Submarine/Submarine.less [hunk 1]
```diff
@@ -146,14 +149,14 @@
 		}
 
 		& > .lucid-Submarine-Primary {
-			flex-basis: 0px;
+			flex-basis: 0;
```
Comment (ogupte) [inline src/components/Submarine/Submarine.less]: flex-basis was set to `0px` to fix an IE bug where it doesn't treat `0` and `0px` the same. So we might need to ignore the lint rules for this line.

File: src/components/Tabs/Tabs.less [hunk 2]
```diff
@@ -17,7 +17,7 @@
 	}
 
 	&-Tab {
-		flex: 1 1 0px;
+		flex: 1 1 0;
```
Comment (ogupte) [inline src/components/Tabs/Tabs.less]: same IE bug here with flex-basis being `0px`

File: src/components/Submarine/Submarine.less (hunk 1 above)
Comment (ogupte) [inline src/components/Submarine/Submarine.less]: https://connect.microsoft.com/IE/feedback/details/949764/ie-11-incorrectly-rejects-flexbox-shorthand-syntax-flex-1-1-0-as-invalid

File: src/styles/components.less [hunk 3]
```diff
@@ -29,7 +29,6 @@
 @import '../components/Icon/ChevronIcon/ChevronIcon';
 @import '../components/Icon/DangerIcon/DangerIcon';
 @import '../components/Icon/DangerLightIcon/DangerLightIcon';
-@import '../components/Icon/EditIcon/EditIcon';
```
Comment (jondlm) [inline src/styles/components.less]: Was this supposed to be removed?

Comment (alisonailea) [inline src/styles/components.less]: EditIcon.less was an empty file so I removed it

File: src/components/Submarine/Submarine.less (hunk 1 above)
Comment (alisonailea) [inline src/components/Submarine/Submarine.less]: updated

**Pull request merged**